src/Kdyby/Console/DI/ConsoleExtension.php

```diff
@@ -82,18 +82,16 @@ public function loadConfiguration()
 		Nette\Utils\Validators::assert($config, 'array');
 		foreach ($config['commands'] as $i => $command) {
 			$def = $builder->addDefinition($this->prefix('command.' . $i));
-			list($def->factory) = Nette\DI\Compiler::filterArguments(array(
-				is_string($command) ? new Statement($command) : $command
-			));
-
-			if (class_exists($def->factory->entity)) {
-				$def->class = $def->factory->entity;
-			}
-
-			$def->setAutowired(FALSE);
-			$def->setInject(FALSE);
```

Comment:
We can discuss if it's needed or not, but I would like to leave it out of the scope of this PR, as I like the parsing the service using the "smart way".